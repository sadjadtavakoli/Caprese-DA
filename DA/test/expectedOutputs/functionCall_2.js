
#1 function callee-test/unit_tests/functionCall_2.js-1-5 entered from arrowFunction-test/unit_tests/functionCall_2.js-1-7
#1 function callee-test/unit_tests/functionCall_2.js-1-5 exited to function arrowFunction-test/unit_tests/functionCall_2.js-1-7
#end Execution