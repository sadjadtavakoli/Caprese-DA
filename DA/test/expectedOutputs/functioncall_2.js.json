{"callee-test/unit_tests/functioncall_2.js-1-5":{"impacted":["t_1"]},"keyMap":{"t_1":"test/unit_tests/functioncall_2.js","t_3":"callee-test/unit_tests/functioncall_2.js-1-5"}}